ishare/isharedecider_test.go

```diff
@@ -134,32 +134,28 @@ func TestDecide(t *testing.T) {
 		{"PUT request with AR error should bubble.", getIShareDSBAToken(), "/ngsi-ld/v1/entities/urn:ngsi-ld:entity:to-put", "PUT", nil, "myPdp", model.DelegationEvidence{}, getBadGatewayError(), model.Decision{}, getBadGatewayError(), []model.Policy{getPolicy("entity", []string{"urn:ngsi-ld:entity:to-put"}, []string{"*"}, "PUT")}},
 		{"PATCH request with AR error should bubble.", getIShareDSBAToken(), "/ngsi-ld/v1/entities/urn:ngsi-ld:entity:to-patch", "PATCH", getEntity(), "myPdp", model.DelegationEvidence{}, getBadGatewayError(), model.Decision{}, getBadGatewayError(), []model.Policy{getPolicy("entity", []string{"urn:ngsi-ld:entity:to-patch"}, []string{"id", "myProp"}, "PATCH")}},
 
-		{"Non-NGSI GET requests are considered a BadRequest.", getDSBAToken(), "/non/ngsi/request", "GET", nil, "myPdp", getActivePermit(), model.HttpError{}, model.Decision{Decision: false}, model.HttpError{Status: 400}, []model.Policy{}},
```

Comment:
Why not use one of the paths to check the "http path" case?

Reply:
I will add the tests, but it was to much for now, since the tests matches the full requested policy, wich is quite lengthy;)